firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -83,6 +83,10 @@ private synchronized void executeAllListenerCallbacks(ConfigUpdate configUpdate)
     }
   }
 
+  private synchronized Set<ConfigUpdateListener> getEventListeners() {
```

Comment:
Following a standard getter pattern here seemed to make it easier to read at the callsite since you don't need to visit the method to know what it's doing. But don't feel strongly either way.

Reply:
The pattern makes sense but I think we should still return a `boolean` in the method b/c it's only within that method is the call to `eventListeners` thread-safe. Once the Set is returned back to `handleNotifications` it's not thread safe anymore

Reply:
Good point. Updated.